lib/engine/tile.rb

```diff
@@ -73,15 +73,25 @@ def self.part(type, params, cache)
 
         Part::Path.new(params['a'], params['b'])
       when 'city'
-        city = Part::City.new(params['revenue'], params.fetch('slots', 1), params['groups'], params['hide'])
+        city = Part::City.new(params['revenue'],
+                              params.fetch('slots', 1),
+                              params['groups'],
+                              params['hide'],
+                              params.fetch('movement_cost', 1))
```

Comment:
there's no reason to fetch because you have a default already

Reply:
unfortunately you do need the fetch, I've tried it without and it just sets all the visit costs to zero.
Presumably because you have an argument it'll take it even if it's nil.